content/blog/setting-up-harbor-as-a-local-container-registry-in-hpe-private-cloud-ai.md

```diff
@@ -0,0 +1,304 @@
+pod/nginx-chart-546476cd99-2nqzz   1/1     Running   0          6s
+
+NAME                  TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)   AGE
+service/nginx-chart   ClusterIP   10.99.78.114   <none>        80/TCP    6s
+
+NAME                          READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/nginx-chart   1/1     1            1           6s
+
+NAME                                     DESIRED   CURRENT   READY   AGE
+replicaset.apps/nginx-chart-546476cd99   1         1         1       6s
+```
+
+Within the *'nginx'* namespace , a Secret named *'harbor'*, of type *dockerconfigjson*, is created. This secret is used to authenticate and pull images from the *Harbor* registry during the deployment of the *CFE Nginx* application:
+
+```shell
+# kubectl get secret harbor -n nginx
+NAME     TYPE                             DATA   AGE
+harbor   kubernetes.io/dockerconfigjson   1      3m41s
+```
+Type the following command to observe the *cfe-nginx* image, tagged *v0.1.0*, being pulled from the *demo* private project in *Harbor* registry:
+
+```shell
+[root@ez-master01 ~]# k describe pod/nginx-chart-546476cd99-2nqzz -n nginx
+Name:             nginx-chart-546476cd99-2nqzz
+Namespace:        nginx
+...
+...
+Events:
+  Type    Reason     Age    From                         Message
+  ----    ------     ----   ----                         -------
+  Normal  Scheduled  2m18s  scheduler-plugins-scheduler  Successfully assigned nginx/nginx-chart-546476cd99-2nqzz to scs04.pcai0104.ld7.hpecolo.net
+  Normal  Pulling    2m18s  kubelet                      Pulling image "busybox"
+  Normal  Pulled     2m17s  kubelet                      Successfully pulled image "busybox" in 860ms (860ms including waiting)
+  Normal  Created    2m17s  kubelet                      Created container web-content
+  Normal  Started    2m17s  kubelet                      Started container web-content
+  Normal  Pulling    2m16s  kubelet                      Pulling image "harbor.ingress.pcai0104.ld7.hpecolo.net/demo/cfe-nginx:v0.1.0"
+  Normal  Pulled     2m16s  kubelet                      Successfully pulled image "harbor.ingress.pcai0104.ld7.hpecolo.net/demo/cfe-nginx:v0.1.0" in 377ms (377ms including waiting)
+  Normal  Created    2m16s  kubelet                      Created container nginx-chart
+  Normal  Started    2m16s  kubelet                      Started container nginx-chart 
+```
+The *Logs* page of the *Harbor* UI provides a comprehensive audit trail, capturing key activities such as project and user creation, as well as image push and pull operations: 
+
+![](/img/harbor-audit.png)
+
+## Conclusion
+
+In this blog post, we explored how to deploy *Harbor* to HPE Private Cloud AI and configure it as a local container registry. By setting up a private *Harbor* project and assigning user roles, organizations can securely manage, push and pull container images tailored to their application needs. 
```

Comment:
oh! You were doing so well! 
Please rewrite this to "In this blog post, I explored how to deploy...